iphone/Classes/TiUtils.m

```diff
@@ -112,6 +112,18 @@ + (BOOL)isRetinaiPhoneX
   return (mainScreenBoundsSize.height == 812 || mainScreenBoundsSize.width == 812);
 }
 
++ (BOOL)isRetinaiPhoneXSMax
+{
+  CGSize mainScreenBoundsSize = [[UIScreen mainScreen] bounds].size;
+  return (mainScreenBoundsSize.height == 896 || mainScreenBoundsSize.width == 896) && [TiUtils isRetinaHDDisplay];
```

Comment:
The naming of isRetinaHDDisplay is confusing here for me (and the next method). Presumably `[TiUtils isRetinaHDDisplay]` is equivalent to asking if it's a 3x density (458pp)?

Maybe rename TiUtils.isRetinaHDDisplay to something like `TiUtils.is3xDensity` or `TiUtils.is458ppDensity`? And then below (iPhone XR) checking that we're at 2x density rather than checking we're *not* at 3x?

Reply:
+1 for the 3xDensity

Reply:
Can we add the new name and have the old redirect to the new name and deprecate (the old)?

Reply:
Yep!